Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationEmail.cshtml

```diff
@@ -39,7 +40,7 @@
                             </label>
                         </h1>
 
-                        <div id="hint" class="govuk-hint">Where possible, use a shared inbox that other team members can access</div>
+                        <div id="hint" class="govuk-hint">@StaticTextResource.OrganisationRegistration_EnterOrganisationEmail_Hint</div>
 
                         <div class="govuk-form-group govuk-!-width-two-thirds @(emailAddressHasError ? "govuk-form-group--error" : "")">
                             @if (emailAddressHasError)
```

Comment:
There's a few instances of `<span class="govuk-visually-hidden">Error:</span>` that aren't translated

Reply:
Done